master/static/srv/proto_get_trial_workloads.sql

```diff
@@ -0,0 +1,62 @@
+WITH validations_vt AS (
+  SELECT row_to_json(r1) AS validation,
+    prior_batches_processed
+  FROM (
+      SELECT 'STATE_' || v.state as state,
+        v.start_time,
+        v.end_time,
+        s.num_batches,
+        s.prior_batches_processed,
+        v.metrics->'num_inputs' as num_inputs,
+        v.metrics->'validation_metrics' as metrics
+      FROM validations v
+        INNER JOIN steps s ON v.trial_id = s.trial_id
+        AND v.step_id = s.id
+      WHERE v.trial_id = $1
+    ) AS r1
+),
+trainings_vt AS (
+  SELECT row_to_json(r1) AS training,
+    prior_batches_processed
+  FROM (
+      SELECT s.start_time,
+        s.end_time,
+        'STATE_' || s.state as state,
+        s.num_batches,
+        s.prior_batches_processed,
+        s.metrics->'avg_metrics' as metrics,
+        s.metrics->'num_inputs' as num_inputs
+      FROM steps s
+      WHERE s.trial_id = $1
+    ) AS r1
+),
+checkpoints_vt AS (
```

Comment:
non-blocking: what does `vt` stand for? can these just be trainings, checkpoints, valdiations instead.

Reply:
trainings, checkpoints, valdiations instead

i was trying to avoid reusing actual table names. vt for virtual_table which is probably incorrect. what are these called? views? cve?

Reply:
cte, "common table expression". now that I know what vt is, it make more sense. why not checkpoint_workloads or something?

Reply:
why not checkpoint_workloads or something?
yeah that's even better